libraries/AP_Math/crc.cpp

```diff
@@ -334,3 +334,16 @@ uint32_t crc_crc24(const uint8_t *bytes, uint16_t len)
     }
     return crc;
 }
+
+// simple 8 bit checksum used by FPort
+uint8_t crc_sum8(const uint8_t *p, uint8_t len)
```

Comment:
Probably not a crc as such

Reply:
not, but sits in same class of fns, so I think it fits